### DISCUSSION TITLE
ifnot (is like ‘if’)

### DISCUSSION THREAD

**Discussion by stureandersen** [NilsSve/DfRefactor#44] [Ideas]
Treat IFNOT command like IF

**Comment by NilsSve** [comment]
Does that mean that the line:
`    IFNOT Found Begin`
should be refactored to:
`    IF (Not(Found)) Begin`

Is that true?

**Reply by stureandersen** [reply]
yes, that's right

**Reply by NilsSve** [reply]
OK, thank you. It has been added to the suggestion list for new refactoring functions.

**Comment by NilsSve** [comment]
Closed.
